src/test/java/hw2/FirstExerciseTest.java

```diff
@@ -0,0 +1,86 @@
+package hw2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FirstExerciseTest extends TestBase {
+
+    private static final int EXPECTED_BENEFITS_COUNT = 4;
+    private static final List<String> EXPECTED_BENEFIT_TEXTS = new ArrayList<String>() {{
+        add("To include good practices\nand ideas from successful\nEPAM project");
+        add("To be flexible and\ncustomizable");
+        add("To be multiplatform");
+        add("Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…");
+    }};
+    private static final List<String> EXPECTED_HEADER_ITEMS_TEXT = new ArrayList<String>() {{
+        add("HOME");
+        add("CONTACT FORM");
+        add("SERVICE");
+        add("METALS & COLORS");
+    }};
+    private static final List<String> EXPECTED_SIDE_NAV_BAR_ITEMS_TEXT = new ArrayList<String>() {{
+        add("Home");
+        add("Contact form");
+        add("Service");
+        add("Metals & Colors");
+        add("Elements packs");
+    }};
+
+    private static final String SIDE_NAVIGATION_BAR_NAME_SELECTOR = "navigation-sidebar";
```

Comment:
Do not create constants for locators, use in directly in method calls, remove all of them

Reply:
done